mgmt/RecordsConfig.cc

```diff
@@ -1172,11 +1173,9 @@ static const RecordElement RecordsConfig[] =
   //# Configuration for TLSv1.3 and above
   //#
   //##############################################################################
-  // The default value (nullptr) means the default value of TLS stack will be used.
-  // - e.g. OpenSSL : "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256"
-  {RECT_CONFIG, "proxy.config.ssl.server.TLSv1_3.cipher_suites", RECD_STRING, nullptr, RECU_RESTART_TS, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
+  {RECT_CONFIG, "proxy.config.ssl.server.TLSv1_3.cipher_suites", RECD_STRING, "TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384", RECU_RESTART_TS, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
It doesn't make sense to prefer CHACHA on the server side since most servers will have AES support build into their CPUs and will be faster.

Reply:
Benchmarked on my Mac and AES is ~6 times faster:
```
15:30:36 C02XF1ULJG5J:(std_array)~/dev/apache/trafficserver$ openssl speed aes-128-gcm chacha20-poly1305
Doing aes-128 gcm for 3s on 16 size blocks: 40891550 aes-128 gcm's in 2.98s
Doing aes-128 gcm for 3s on 64 size blocks: 33355414 aes-128 gcm's in 2.97s
Doing aes-128 gcm for 3s on 256 size blocks: 19007676 aes-128 gcm's in 2.99s
Doing aes-128 gcm for 3s on 1024 size blocks: 6378002 aes-128 gcm's in 2.99s
Doing aes-128 gcm for 3s on 8192 size blocks: 803093 aes-128 gcm's in 2.97s
Doing chacha20 poly1305 for 3s on 16 size blocks: 6373621 chacha20 poly1305's in 2.98s
Doing chacha20 poly1305 for 3s on 64 size blocks: 6446820 chacha20 poly1305's in 2.98s
Doing chacha20 poly1305 for 3s on 256 size blocks: 2987367 chacha20 poly1305's in 2.97s
Doing chacha20 poly1305 for 3s on 1024 size blocks: 1019862 chacha20 poly1305's in 2.98s
Doing chacha20 poly1305 for 3s on 8192 size blocks: 143181 chacha20 poly1305's in 3.00s
```